### PR TITLE
General fixes and features

### PR DESCRIPTION
**Pull request by Alphaegen**
- Swapped resume/pause messages because of inverted order
- Added try/catch to lyrics button to prevent crash with Cloudflare captcha

Also reformatted some of the code. Fixed usage of spacing.

### CODE REVIEW

**Comment by Alphaegen** [comment]
Also added a history command now, by simply using the history from the `discord-player`.

**Comment by Alphaegen** [comment]
- Added `@distube/ytdl-core` as the default, since it is preferred over `play-dl`. 
- Also made it usable without needing a .env, which would prevent any `Could not extract stream for this track` errors from occurring no matter the system. (they still occurred on servers with `play-dl` if no .env was used)
- Added a debugging command (might not be useful now, but probably will be later.

File: main.js
Comment (revot334) [inline main.js:1]: please remove debugging

File: package.json
Comment (revot334) [inline package.json:1]: remove not need with yarn

**Review by revot334 — CHANGES_REQUESTED** [review]
please see comments

File: main.js
Comment (Alphaegen) [reply main.js:1]: Debugging has been removed

File: src/buttons/lyrics.js
Comment (dragiez) [inline src/buttons/lyrics.js:1]: where have u defined genius?

**Comment by revot334** [comment]
please fix node start script dose not work

**Pull request merged**